sdk/communication/Azure.Communication.Rooms/README.md

```diff
@@ -52,8 +50,99 @@ RoomsClient client = new RoomsClient(connectionString);
 ```
 
 ## Examples
+### Create a room
+To create a room, call the `CreateRoom` or `CreateRoomAsync` function from `RoomsClient`.
+The `validFrom`, `validUntil`, and `participants` arguments are all optional. If `validFrom` and `validUntil` are not provided, then the default for `validFrom` is current date time and the default for `validUntil` is `validFrom + 180 days`.
+When defining `InvitedRoomParticipants`, if role is not specified, then it will be `Attendee` by default.
+The returned value is `Response<CommunicationRoom>` which contains created room details as well as the status and associated error codes in case of a failure.
+
+```C# Snippet:Azure_Communication_Rooms_Tests_Samples_CreateRoomAsync
+// Create communication users using the CommunicationIdentityClient
+var communicationUser1 = communicationIdentityClient.CreateUserAsync().Result.Value;
+var communicationUser2 = communicationIdentityClient.CreateUserAsync().Result.Value;
+
+var validFrom = DateTimeOffset.UtcNow;
+var validUntil = validFrom.AddDays(1);
+InvitedRoomParticipant participant1 = new InvitedRoomParticipant(communicationUser1); // If role is not provided, then it is set as Attendee by default
+InvitedRoomParticipant participant2 = new InvitedRoomParticipant(communicationUser2) { Role = ParticipantRole.Presenter};
+List<InvitedRoomParticipant> invitedParticipants = new List<InvitedRoomParticipant>
+{
+    participant1,
+    participant2
+};
+
+Response<CommunicationRoom> createRoomResponse = await roomsClient.CreateRoomAsync(validFrom, validUntil, invitedParticipants);
+CommunicationRoom createCommunicationRoom = createRoomResponse.Value;
+```
+
+### Update a room
+The `validFrom` and `validUntil` properties of a created room can be updated by calling the `UpdateRoom` or `UpdateRoomAsync` function from `RoomsClient`.
+
+```C# Snippet:Azure_Communication_Rooms_Tests_Samples_UpdateRoomAsync
+validUntil = validFrom.AddDays(30);
+Response<CommunicationRoom> updateRoomResponse = await roomsClient.UpdateRoomAsync(createdRoomId, validFrom, validUntil);
+CommunicationRoom updateCommunicationRoom = updateRoomResponse.Value;
+```
+
+### Get a created room
+A created room can be retrieved by calling the `GetRoom` or `GetRoomAsync` function from `RoomsClient` and passing in the associated `roomId`.
+
+```C# Snippet:Azure_Communication_Rooms_Tests_Samples_GetRoomAsync
+Response<CommunicationRoom> getRoomResponse = await roomsClient.GetRoomAsync(createdRoomId);
+CommunicationRoom getCommunicationRoom = getRoomResponse.Value;
+```
+
+### List all created rooms
```

Comment:
Is this a todo? Please either add the snippet or remove this subsection for now.

Reply:
Removed for the time being, thanks!